docs/requirements.txt

```diff
@@ -21,5 +21,6 @@ filelock
 prettytable
 psutil
 ruamel.yaml
+dill
```

Comment:
what is this package used for?

Reply:
Doc test in pipeline suggests installing this package.